util/maven/generate_poms.py

```diff
@@ -58,6 +58,29 @@ def pom_deps(label):
         'name': 'Dagger',
         'artifact': 'dagger',
     },
+    '//gwt:gwt': {
+        'name': 'Dagger GWT',
+        'artifact': 'dagger-gwt',
+        'manual_dependencies': '''
```

Comment:
If you think that is worth a follow-up to this change, that's also totally valid.